src/library/scala/collection/StringOps.scala

```diff
@@ -1594,17 +1594,19 @@ final class StringOps(private val s: String) extends AnyVal {
   def sliding(size: Int, step: Int = 1): Iterator[String] = new WrappedString(s).sliding(size, step).map(_.unwrap)
 
   /** Iterates over combinations.  A _combination_ of length `n` is a subsequence of
-    *  the original string, with the chars taken in order.  Thus, `"xy"` and `"yy"`
-    *  are both length-2 combinations of `"xyy"`, but `"yx"` is not.  If there is
+    *  the original string, with the chars taken in order of its first occurrence.  Thus, `"yy"` and `"yx"`
```

Comment:
```suggestion
    *  the original string, with the chars taken in order of their first occurrence.  Thus, `"yy"` and `"yx"`
```

This one needs to be fixed too